cognite/client/_api/transformation_jobs.py

```diff
@@ -0,0 +1,103 @@
+from typing import List, Optional
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import TransformationJob, TransformationJobList, TransformationJobMetricList
+
+
+class TransformationJobsAPI(APIClient):
+    _RESOURCE_PATH = "/transformations/jobs"
+    _LIST_CLASS = TransformationJobList
+
+    def list(self, limit: Optional[int] = 25, transformation_id: Optional[int] = None) -> TransformationJobList:
+        """`List all running transformation jobs. <https://docs.cognite.com/api/playground/#operation/transformationJobs>`_
```

Comment:
please refrain from using playground, we really want to move away from playground.

Reply:
this hyperlink doesn't point anywhere. guess you need to change it to https://github.com/cognitedata/service-contracts/pull/1501/files#diff-8df5cf6e84a989d49d837ea04347ac6085957197f0ef01467745af40090de15dR1028

but the link above points to a v1 endpoint, so we should point this there too.

Also, the operation Id is a bit lengthy and cumbersome to read. Is there a way to shorten it? have a look at other op ids for example

Reply:
my bad, the API is already in V1, but forgot to update the links in the docstrings